README.markdown

```diff
@@ -26,6 +26,12 @@ Create the sample topic. By default, the topic will have 8 partitions:
 $ heroku kafka:topics:create messages
 ```
 
+Create the consumer group:
```

Comment:
I believe this is only necessary for `basic-*` plans, right? If so, perhaps that should be noted here.

Reply:
I added a note. WDYT?